lib/src/common/syntax.dart

```diff
@@ -1,28 +1,34 @@
 part of guinness2;
 
 void beforeEach(Function fn, {int priority: 0}) =>
-    guinness._context.addBeforeEach(fn, priority: priority);
+    dartTest.setUp(fn);
 
 void afterEach(Function fn, {int priority: 0}) =>
-    guinness._context.addAfterEach(fn, priority: priority);
-
-void it(name, [Function fn]) => guinness._context.addIt(name.toString(), fn,
-    excluded: false, exclusive: false);
+    dartTest.tearDown(fn);
+
+void it(name, [Function fn]) {
+  if (fn != null) {
+    dartTest.test(name, fn);
+  } else {
+    dartTest.test(name, () {}, skip: 'skipping emtpy it');   
```

Comment:
Oh, thanks! I kind of like having the cause of skipping listed, so I'll keep the message for now, but that's good to know.